### PR TITLE
feat follow-unfollow

### PR DESCRIPTION
**Pull request by deepak29-git**
user can following / unfollow

image

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
code structure is good .

File: src/all-api/comment-api.js [hunk 1]
```diff
@@ -6,22 +6,22 @@ console.log(getToken())
...
+      // const {data}=await axios.post(`/api/comments/add/${postId}`,
+      // {
+      //   commentData:commentInput
+      // },
+      // {
+      //     headers:{
+      //         authorization:getToken()
+      //     }
+      // }
+      // )
```
Comment (tushargoyal812) [inline src/all-api/comment-api.js:18]: remove comments

File: src/all-api/comment-api.js [hunk 2]
```diff
@@ -6,22 +6,22 @@ console.log(getToken())
...
+    const { data } = await axios({
+      method: "POST",
+      url: `/api/comments/add/${postId}`,
+      data: {commentData: {id:postId,text:commentInput} },
+      headers: { authorization: getToken() },
+    });
     console.log(data);
```
Comment (tushargoyal812) [inline src/all-api/comment-api.js:25]: remove console

File: src/all-api/follow-unfollow-api.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+const followUserApi = async (followUserId, userDispatch) => {
+  try {
+    const { data } = await axios({
+      method: "POST",
+      url: `/api/users/follow/${followUserId}`,
+      data: {},
+      headers: { authorization: getToken() },
+    });
+    console.log(data.user)
```
Comment (tushargoyal812) [inline src/all-api/follow-unfollow-api.js:12]: remove console

**Review by tushargoyal812 — COMMENTED** [review]
file structure is good.
variable naming is good.
design looks cool.

File: src/all-api/follow-unfollow-api.js (hunk 3 above)
Comment (deepak29-git) [reply src/all-api/follow-unfollow-api.js:12]: ok

File: src/all-api/comment-api.js (hunk 2 above)
Comment (deepak29-git) [reply src/all-api/comment-api.js:25]: ok

File: src/all-api/comment-api.js (hunk 1 above)
Comment (deepak29-git) [reply src/all-api/comment-api.js:18]: ok

**Pull request merged**